openedx_lti_tool_plugin/utils.py

```diff
@@ -107,3 +110,25 @@ def get_pii_from_claims(claims: dict) -> dict:
         'family_name': claims.get('family_name', ''),
         'locale': claims.get('locale', ''),
     }
+
+
+def validate_context_key(context_key: str):
+    """Validate context key is a CourseKey or UsageKey.
+
+    Args:
+        context_key: CourseKey or UsageKey string.
+    """
+    invalid_course_key = False
+    invalid_usage_key = False
+
+    try:
+        CourseKey.from_string(context_key)
+    except InvalidKeyError:
+        invalid_course_key = True
+    try:
+        UsageKey.from_string(context_key)
+    except InvalidKeyError:
+        invalid_usage_key = True
+
+    if invalid_course_key and invalid_usage_key:
+        raise ValidationError(_(f'Invalid context key: {context_key}. Should be either a CourseKey or UsageKey'))
```

Comment:
```suggestion
        raise ValidationError({
            'context_key': _(f'Invalid context key: {context_key}. Should be either a CourseKey or UsageKey'),
        })
```
What do you think about adding the field to the validation error?